docs/diagrams/ArchitectureSequenceDiagram.puml

```diff
@@ -14,7 +14,7 @@ activate ui UI_COLOR
 ui -[UI_COLOR]> logic : execute("delete 1")
 activate logic LOGIC_COLOR
 
-logic -[LOGIC_COLOR]> model : deleteInternApplication(i)
+logic -[LOGIC_COLOR]> model : deleteInternApplication(0)
```

Comment:
Wait, isn't `i` the correct one? 
![image](https://github.com/AY2324S1-CS2103T-W17-2/tp/assets/107602488/f4eeaeb5-0089-4b64-bfae-1deb853a0f36)
According to this picture from the `ModelManager` class.

Reply:
Just an added comment, I think it might be safer to put as deleteInternApplication(internApplication) instead of using `i`

Reply:
Noted, thanks You Wei!